shiny/ui/_chat.py

```diff
@@ -406,7 +406,7 @@ def messages(
     def messages(
         self,
         *,
-        format: MISSING_TYPE | ProviderMessageFormat = MISSING,
+        format: ProviderMessageFormat | Literal["internal"],
```

Comment:
Was it intentional to require a value here? I'd prefer if it wasn't necessary since the `"internal"` format works for most formats

```suggestion
        format: ProviderMessageFormat | Literal["internal"] = "internal",
```

Reply:
Oh wait, I think I misunderstood what the default format was -- I thought it was the actual internal format that the `Chat` class uses, but that's not right. The internal format is `StoredMessage`, not `ChatMessage`. That's why I thought it wouldn't make sense to return that format by default.

So to summarize, prior to this PR, the default format was `ChatMessage`, which is sort of a lowest-common-denominator format that has `role` and `content`. 

So it might make sense to just drop this PR entirely. That said, it could still be useful to be able to access the `StoredMessage`s in the chat object. That's something I actually wanted to do when I was trying to get a handle on the sizes of messages being sent to the LLM (but then I got sidetracked before I went in depth on it).

Reply:
Ah, I see. I'd be interested to know more about why you want access to `StoredMessage`.

I'll close this PR and create another one for the `format="anthropic"` thing